sdk/keyvault/keyvault-keys/samples/javascript/cryptography.js

```diff
@@ -28,7 +28,9 @@ async function main() {
   // Connection to Azure Key Vault Cryptography functionality
   let myWorkKey = await client.createKey(keyName, "RSA");
 
-  const cryptoClient = new CryptographyClient(myWorkKey.id, credential);
+  const cryptoClient = new CryptographyClient(
+    myWorkKey.id // Or just `myWorkKey`. You can use either the key or the key Id to create a CryptographyClient.
```

Comment:
When one has access to the whole key, there is no reason for them to use `key.id` than `key` directly here right?

Is it common for a user to get the id from somewhere else and use it? Or is it common for user to get/create the key, then get the id, then use it?

If it is the former, then to keep things simple, I would recommend using the overload that uses the key directly in the sample and then say that you can also pass the key id which is its url directly as well

For example:

```suggestion
    myWorkKey // You can also pass the identifier of the key i.e it's url here
```

Reply:
It is common to use the ID directly. I like your proposed changes though! Appreciated.